presto-spi/src/main/java/com/facebook/presto/spi/connector/ConnectorCommitHandle.java

```diff
@@ -13,6 +13,14 @@
  */
 package com.facebook.presto.spi.connector;
 
+import com.facebook.presto.spi.SchemaTableName;
+
+import java.util.Collections;
+
 public interface ConnectorCommitHandle
 {
+    default Object getLastDataCommitTimes(SchemaTableName table)
```

Comment:
Why are we using `Object` instead of `List<DateTime>` ? Is it to avoid adding dependency on `joda-time `for `presto-spi` module ?

Reply:
@highker, ConnectorCommitHandle will be put into QueryOutputMetadata, which contains the schema and table name, which can be used to form the SchemaTableName object. This will be called in PrismEventListener before we log.